OrbitQt/main.cpp

```diff
@@ -154,7 +174,8 @@ static void StyleOrbit(QApplication& app) {
   darkPalette.setColor(QPalette::HighlightedText, Qt::black);
   app.setPalette(darkPalette);
   app.setStyleSheet(
-      "QToolTip { color: #ffffff; background-color: #2a82da; border: 1px solid "
+      "QToolTip { color: #ffffff; background-color: #2a82da; border: 1px "
+      "solid "
```

Comment:
nit: I think this got auto formatted when it was indented deeper.

Reply:
fixed. thanks.